apps/fluent-tester/src/FluentTester/TestComponents/FocusTrapZone/consts.ts

```diff
@@ -3,8 +3,6 @@ export const FOCUSTRAPZONE_TESTPAGE = 'FocusTrapZone_TestPage';
 
 /* E2E Testing FocusTrapZone 1 */
 export const FOCUSTRAPZONE_TEST_COMPONENT = 'FocusTrapZone_Test_Component'; // A component on each specific test page
-export const FOCUSTRAPZONE_ACCESSIBILITY_LABEL = 'E2E testing FocusTrapZone_Test_Component accessibility label';
 
 /* E2E Testing FocusTrapZone 2 */
 export const FOCUSTRAPZONE_NO_A11Y_LABEL_COMPONENT = 'FocusTrapZone_No_A11y_label_Component';
```

Comment:
Are these constants used? `FOCUSTRAPZONE_NO_A11Y_LABEL_COMPONENT`, `FOCUSZONE_NO_A11Y_LABEL_COMPONENT`, and `SEPARATOR_NO_A11Y_LABEL_COMPONENT`?

Reply:
Not quite yet. I preloaded them in because they will be used in the near feature once I add more functional tests. I can, however, change the name, because it doesn't reflect properly what each component will be